### PR TITLE
Added Forms EditorRenderer and Editor sample

### PR DESCRIPTION
**Pull request by jsuarezruiz**
Added basic **Editor** control support.

ooui-forms-editor

Current **features**:

- Text Property
- TextColor Property
- Fonts Properties
- TextChanged Event

Also, added a basic sample.

### CODE REVIEW

File: Ooui.Forms/Renderers/EditorRenderer.cs [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    public class EditorRenderer : ViewRenderer<Editor, Ooui.TextArea>
+    {
+        private bool _disposed;
+        private Ooui.Color _defaultTextColor;
+
+        static Size initialSize = Size.Zero;
+
+        protected IElementController ElementController => Element as IElementController;
+
+        public override SizeRequest GetDesiredSize(double widthConstraint, double heightConstraint)
```
Comment (praeclarum) [inline Ooui.Forms/Renderers/EditorRenderer.cs]: Please use the .editorconfig file so that your formatting matches the project's.

**Comment by praeclarum** [comment]
Ah shoot, looks like we duplicated each other's work! I do not need the EditorRenderer anymore, but would still love the example. Would you mind rebasing to master and using my EditorRenderer?

File: Ooui.Forms/Renderers/EditorRenderer.cs (hunk 1 above)
Comment (jsuarezruiz) [reply Ooui.Forms/Renderers/EditorRenderer.cs]: Ok!

**Comment by jsuarezruiz** [comment]
Oh, I just saw it. You're right!. I have obtained the latest changes. Your EditorRenderer is used but I have kept the Editor sample :)

**Comment by praeclarum** [comment]
Thank you!

**Pull request merged**